src/Components/TwoOneOneComponents/TwoOneOneHeader/TwoOneOneHeader.css

```diff
@@ -9,12 +9,23 @@
   font-size: 13px;
 }
 
+.twoOneOne-globe-icon {
+  color: var(--primary-color);
+}
+
+.twoOneOne-header-container {
+  background-color: #ffffff;
+  height: 5rem;
+  padding: 0 0.5rem;
+  width: 100%;
+}
+
 #twoOneOne-select-language {
   padding: 0;
   color: #005191;
   font-weight: 600;
-  font-size: 0.79rem;
-  margin-left: 13px;
+  font-size: 13px;
```

Comment:
We usually use `rem` for font size, because 211 has a different font size. This component is ok to be `px`, because it is only rendered for 211, but what do you think about using `rem` for constancy?

Reply:
Absolutely!